src/gilded_rose.spec.js

```diff
@@ -1,11 +1,149 @@
 import { Item, updateQuality } from './gilded_rose';
 
-describe('`updateQuality`', () => {
-  it("Updates the Quality", () => {
-    const standardItem = new Item('Haunted Shoe', 10, 10);
-    updateQuality([standardItem]);
-    expect(standardItem.quality).toBe(9);
+describe('updating of aged brie', () => {
+  const agedBrie = new Item('Aged Brie', 2, 0);
+
+  afterEach(() => {
+    // updates the quality and sell_in back to original quantities
+    agedBrie.sell_in = 2;
+    agedBrie.quality = 0;
+  });
+
+  it('increases the quality of the item by 1', () => {
+    updateQuality([agedBrie]);
+    expect(agedBrie.quality).toBe(1);
+  });
+
+  // it('decreases the sell_in of the item by 1', () => {
+  //   updateQuality([agedBrie]);
+  //   expect(agedBrie.sell_in).toBe(1);
+  // });
+
+  it('does not increase the quality of the item to more than 50', () => {
+    agedBrie.quality = 50;
+    updateQuality([agedBrie]);
+    expect(agedBrie.quality).toBe(50);
+  });
+
+  it('increases in quality of the item by 2 if sell_in is less than 0', () => {
+    agedBrie.sell_in = -1;
+    updateQuality([agedBrie]);
+    expect(agedBrie.quality).toBe(2);
+  });
+
+});
+
+describe('updating of sulfuras', () => {
+  const sulfuras = new Item('Sulfuras, Hand of Ragnaros', 0, 80);
+
+  it('does not update sell_in', () => {
+    updateQuality([sulfuras]);
+    expect(sulfuras.sell_in).toBe(0);
+  });
+
+  it('does not update quality', () => {
+    updateQuality([sulfuras]);
+    expect(sulfuras.quality).toBe(80);
+  });
+
+});
+
+describe('updating of conjured items', () => {
+  const conjuredItem = new Item('Conjured Mana Cake', 3, 6);
+
+  afterEach(() => {
```

Comment:
ooooo ok now I see what you were doing with the after Each. I thought you only applied it to the one test before, you made it run for each assertion. That is super dope!!!